source/puppet/5.3/hiera_automatic.md

```diff
@@ -135,71 +140,82 @@ puppet lookup [--help] [--type <TYPESTRING>] [--merge first|unique|hash|deep] [-
-
-* --help: Print a usage message.
-* --explain: Explain the details of how the lookup was performed and where the final value came from, or the reason no value was found. Useful when debugging Hiera data. If `--explain` isn't specified, lookup exits with 0 if a value was found and 1 if not. With `--explain`, lookup always exits with 0 unless there is a major error. You can provide multiple lookup keys to this command, but it only returns a value for the first found key, omitting the rest.
-* --node <NODE-NAME>: Specify which node to look up data for; defaults to the node where the command is run. The purpose of Hiera is to provide different values for different nodes; use specific node facts to explore your data. If the node where you're running this command is configured to talk to PuppetDB, the command will use the requested node's most recent facts. Otherwise, override facts with the '--facts' option.
-* --facts <FILE>: Specify a JSON or YAML file that contains key-value mappings to override the facts for this lookup. Any facts not specified in this file maintain their original value.
-* --environment <ENV>: Specify an environment. Different environments can have different Hiera data.
-* --merge first/unique/hash/deep: Specify the merge behavior, overriding any merge behavior from the data's `lookup_options`.
-* --knock-out-prefix <PREFIX-STRING>: Used with 'deep' merge. Specifies a prefix to indicate a value should be removed from the final result.
-* --sort-merged-arrays: Used with 'deep' merge. When this flag is used, all merged arrays are sorted.
-* --merge-hash-arrays: Used with the 'deep' merge strategy. When this flag is used, hashes within arrays are deep-merged with their counterparts by position.
-* --explain-options: Explain whether a `lookup_options` hash affects this lookup, and how that hash was assembled. (`lookup_options` is how Hiera configures merge behavior in data.)
-* --default <VALUE>: A value to return if Hiera can't find a value in data. Useful for emulating a call to the `lookup function that includes a default.
-* --type <TYPESTRING>: Assert that the value has the specified type. Useful for emulating a call to the `lookup` function that includes a data type.
-* --compile: Perform a full catalog compilation prior to the lookup. If your hierarchy and data only use the `$facts`, `$trusted`, and `$server_facts` variables, you don't need this option. If your Hiera configuration uses arbitrary variables set by a Puppet manifest, you need this to get accurate data. The lookup command doesn’t cause catalog compilation unless this flag is given.
-* --render-as s/json/yaml/binary/msgpack: Specify the output format of the results; `s` means plain text. The default when producing a value is `yaml` and the default when producing an explanation is `s`.
-
-Related topics: [Creating and editing data with Hiera][editing_data]
+The `puppet lookup` command has the following command options.
+
+-   `--help`: Print a usage message.
```

Comment:
Wrapped the command options in preformatting backticks for consistency. This is especially necessary for the tag-like placeholders (such as `<NODE-NAME>`), which might not render correctly otherwise.